templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -23,7 +23,7 @@ jobs:
     steps:
       {{ checkout(depth=0, path=plugin_name) | indent(6) }}
 
-      {{ setup_python(pyversion="3.8") | indent(6) }}
```

Comment:
There's no reason to use a default here.  We should use whatever the template config is set to.